practice-app/apiDocs/home/homePage.yml

```diff
@@ -0,0 +1,124 @@
+Get recent posts of the followings of the given user
+This endpoints It takes a path parameter
```

Comment:
It seems this line didn't get edited. May be a little fix up?

Reply:
You are right. Fixed